README.md

```diff
@@ -144,13 +144,30 @@ sudo rm -rf ~/EspruinoHub
 Run with Docker
 ---------------
 
-Build on Raspberry Pi Zero:
+More information how work Bluetooth in docker you can read in article "[How to run containerized Bluetooth applications with BlueZ](https://medium.com/omi-uulm/how-to-run-containerized-bluetooth-applications-with-bluez-dced9ab767f6)" by Thomas Huffert
 
-    docker build -t espruino/espruinohub:armhf https://github.com/espruino/EspruinoHub.git
```

Comment:
Is it worth leaving this line in just in case someone wants to build it themselves at some point?